prime-router/src/test/kotlin/FileSettingsTests.kt

```diff
@@ -91,7 +94,7 @@ class FileSettingsTests {
                 countyName = null,
                 senders = listOf(),
                 receivers = listOf(
-                    Receiver("elr", "single", "topic", "schema")
+                    Receiver("elr", "single", "topic", "status", "schema")
```

Comment:
use of the enum values, instead of "status"

Reply:
Updated